tests/test.rs

```diff
@@ -72,7 +72,7 @@ fn assert_reference_example(de: fn(&Path, ResolveOptions) -> Result<Config>) ->
     assert_eq!(config.future_incompat_report.frequency, Some(Frequency::Always));
 
     // TODO
-    // [cargo-new]
+    assert_eq!(config.cargo_new.vcs, None);
```

Comment:
We can test this by uncommenting the following two lines:

https://github.com/taiki-e/cargo-config2/blob/bb273c0e03ae844238beabbe712682d7366954b6/tests/fixtures/reference/.cargo/config.toml#L41-L42